docs: Search Metadata Batch 05 (#14621)
hashicorp/consul: website/content/docs/k8s/crds/upgrade-to-crds.mdx

---
layout: docs
page_title: Upgrade Existing Clusters to Use Custom Resource Definitions
description: >-

> Kubernetes clusters configured with a Consul Helm chart version older than 0.30.0 require updates in order to use CRDs. Learn about the changes you need to make, as well as how to migrate a Consul config entry to a k8s CRD.

Reviewer: ```suggestion
Kubernetes clusters configured with a Consul Helm chart version older than 0.30.0 require updates in order to use CRDs. Learn about upgrading to a supported Helm version and how to migrate a Consul config entry to a k8s CRD.
```
